src/deploy/missions/contracts/z-token/z-token.ts

```diff
@@ -86,35 +86,36 @@ IZNSContracts
   }
 
   getArtifact () {
-    return meowArtifact;
+    return zArtifact;
   }
 
+  // it will choose the governon as `admin` argument
+  // and deployAdmin as `minter` and first passed admin as `mintBeneficiary`.
   async deployArgs () : Promise<TDeployArgs> {
-    return [meowTokenName, meowTokenSymbol];
+    const {
+      initialAdminDelay,
+      initialSupplyBase,
+      inflationRates,
+      finalInflationRate,
+    } = this.config.zTokenConfig as unknown as IZTokenConfig;
+    return [
+      zTokenName,
+      zTokenSymbol,
+      this.config.governorAddresses[0],
+      initialAdminDelay,
+      this.config.deployAdmin.address,
+      this.config.adminAddresses[0],
+      initialSupplyBase,
+      inflationRates,
+      finalInflationRate,
+    ];
   }
 
   async needsPostDeploy () {
-    const msg = this.config.mockMeowToken ? "needs" : "doesn't need";
+    const msg = this.config.ZToken ? "needs" : "doesn't need";
 
     this.logger.debug(`${this.contractName} ${msg} post deploy sequence`);
 
-    return this.config.mockMeowToken ;
-  }
-
-  async postDeploy () {
-    const {
-      meowToken,
-      config: {
-        deployAdmin,
-      },
-    } = this.campaign;
-
-    // Mint 100,000 MEOW to the deployer
-    await meowToken.connect(deployAdmin).mint(
```

Comment:
I mentioned re adding a public `mint` function to the mock above, be sure to put this back in place when you do